package.json

```diff
@@ -93,6 +103,20 @@
         "category": "Ledger",
         "tooltip": "Select the device you want to build your app for."
       },
+      {
+        "command": "selectVariant",
```

Comment:
I suggest we only keep the variant selection icon and remove the `Select variant` line. 

We can move the variant selection icon to the `Select app` line and then add the variant next to the app name like `[app-name / variant-name]`. It wil make the UI less cluttered.

Reply:
Not fully agree :wink: 
My opinion is that the Select app line may become too long, because the app displayed is in fact the last part of app dir path.
I prefer a separate line, it is clearer, and easier to read and understand

Reply:
Ok then, let's keep it as a separate line but we don't need the icon at then of the line, it's useless since clicking on the line itself opens the menu :man_shrugging:

Reply:
Ok, removed